clinica/iotools/converters/ukb_to_bids/ukb_utils.py

```diff
@@ -394,3 +396,21 @@ def select_sessions(x: DataFrame) -> Series:
         return x.age_when_attended_assessment_centre_f21003_2_0
     elif x["source_sessions_number"] == "3":
         return x.age_when_attended_assessment_centre_f21003_3_0
+
+
+def import_event_tsv(bids_path: str) -> None:
```

Comment:
I agree on a theorical standpoint, but i really dislike the idea of installing it there, since it is meant to be used only by tfmri data and the tsv would be above not only fmri but the other modalities as well.

That being said, either options is okay with me. wdyt @ghisvail